docs/index.html

```diff
@@ -20,24 +20,24 @@
     />
     <link
       rel="stylesheet"
-      href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css"
+      href="//cdn.jsdelivr.net/npm/docsify@4.12.2/lib/themes/vue.css"
```

Comment:
Why not `@4`?

Reply:
I think @Koooooo-7's idea below, to replace the version automatically, would be great. Let's go with that approach because then it will behave like `@4` for us (for Docsify site), but users will see the full version and will copy/paste that into their projects which is good.